package agent;

message LoginRequest {
	required uint64 account = 1;
	required uint64 userid = 2;
  required string sessionkey = 3;
  required uint32 version = 4;
}


